admin/src/main/scala/za/co/absa/spline/admin/AdminCLI.scala

```diff
@@ -107,7 +110,65 @@ class AdminCLI(dbManagerFactory: ArangoManagerFactory) {
           action { case (_, c@AdminCLIConfig(cmd: DBInit, _, _)) => c.copy(cmd.copy(force = true)) },
         opt[Unit]('s', "skip")
           text "Skip existing database. Don't throw error, just end."
-          action { case (_, c@AdminCLIConfig(cmd: DBInit, _, _)) => c.copy(cmd.copy(skip = true)) })
+          action { case (_, c@AdminCLIConfig(cmd: DBInit, _, _)) => c.copy(cmd.copy(skip = true)) },
+
+        opt[Map[String, Int]]("shard-num")
+          text "Override number of shards per collection. Comma-separated key-value pairs, e.g. 'collectionA=2,collectionB=3'."
+          validate (
+          _.values.collectFirst({
+            case v if v < 1 => failure(s"Shard number should be positive, but was $v")
+          }).getOrElse(success))
+          action {
+          case (m, c@AdminCLIConfig(cmd@DBInit(_, _, _, opts), _, _)) =>
+            c.copy(cmd.copy(options = opts.copy(numShards = m.mapKeys(CollectionDef.forName))))
+        },
```

Comment:
The indentation seems wrong. 

```suggestion
        opt[Map[String, Int]]("shard-num")
          text "Override number of shards per collection. Comma-separated key-value pairs, e.g. 'collectionA=2,collectionB=3'."
          validate (
            _.values.collectFirst({
              case v if v < 1 => failure(s"Shard number should be positive, but was $v")
            }).getOrElse(success))
          action {
            case (m, c@AdminCLIConfig(cmd@DBInit(_, _, _, opts), _, _)) =>
              c.copy(cmd.copy(options = opts.copy(numShards = m.mapKeys(CollectionDef.forName))))
          },
```

Reply:
It's a long standing IDEA auto-layout problem. I tried different settings, but it didn't seem it can do it properly for infix notation in Scala code. So I've switched to prefix notation and layout is now correct.